sdk/python/kubeflow/training/utils/utils.py

```diff
@@ -261,14 +295,21 @@ def get_tfjob_template(
 def get_pytorchjob_template(
     name: str,
     namespace: str,
-    pod_template_spec: models.V1PodTemplateSpec,
+    master_pod_template_spec: models.V1PodTemplateSpec = None,
+    worker_pod_template_spec: models.V1PodTemplateSpec = None,
     num_worker_replicas: Optional[int] = None,
+    num_chief_replicas: Optional[int] = 0,
+    nproc_per_node: Optional[int] = None,
```

Comment:
Let's keep naming consistent everywhere.
```suggestion
    num_procs_per_worker: Optional[int] = None,
```